spras/meo.py

```diff
@@ -46,6 +47,16 @@ def write_properties(filename=Path('properties.txt'), edges=None, sources=None,
         # Do not need csp.phase, csp.gen.file, or csp.sol.file because MAXCSP is not supported
 
 
+"""
+MEO can support partially directed graphs
+
+Expected raw input format:
+Interactor1   pp/pd   Interactor2   Weight
+- the expected raw input file should have node pairs in the 1st and 3rd columns, with a directionality in the 2nd column and the weight in the 4th column
+- it use pp for undirected edges and pd for directed edges
+- it cannot include repeated and bidirectional edges
+- there is a chance MRO assumes that it should just be an undirected edge instead
```

Comment:
```suggestion
- it cannot include repeated and directed edges in opposite directions(A->B and B->A in the input)
- MEO assumes that it should be an undirected edge instead
```
Is this the case you are referring to?